import/index_java/src/org/solrmarc/index/VuFindIndexer.java

```diff
@@ -1444,35 +1444,59 @@ public static String getDeweySortableByType(
         if (list034 != null) {
             for (VariableField vf : list034) {
                 DataField df = (DataField) vf;
-                String d = df.getSubfield('d').getData();
-                String e = df.getSubfield('e').getData();
-                String f = df.getSubfield('f').getData();
-                String g = df.getSubfield('g').getData();
-                //System.out.println("raw Coords: "+d+" "+e+" "+f+" "+g);
-
-                // Check to see if there are only 2 coordinates
-                // If so, copy them into the corresponding coordinate fields
-                if ((d !=null && (e == null || e.trim().equals(""))) && (f != null && (g==null || g.trim().equals("")))) {
-                    e = d;
-                    g = f;
+                // Initialize coordinate variables
+                String d_coord = "";
```

Comment:
(Note that if you follow up on my refactoring suggestions in the next comment, this issue goes away by itself).